src/types/chains-config/MetadataConsts.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractInt } from '@polkadot/types/codec/AbstractInt';
+
+export interface MetadataConstDefinition extends MetadataWeightDefinition {
+	/**
+	 * Runtimes that include their specific definition of extrinsicBaseWeight
+	 */
+	runtimeVersions: number[];
+}
+
+export interface MetadataWeightDefinition {
+	/**
+	 * Polkadot runtime versions before v0.8.27
+	 * Kusama runtime versions before v2027
+	 */
+	extrinsicBaseWeight?: number | AbstractInt;
+	/**
+	 * Polkadot runtime versions after v0.8.26
```

Comment:
0.8.X refers to the client release and all the runtimes in it. Polkadot v26 refers to the specVersion of the polkadot runtime in client release 0.8.26.